### ISSUE TITLE
github changelog generator breaks if no merge sha is found

### ISSUE THREAD

**Issue by rpbaltazar** [CharMixer/auto-changelog-action#6]
```
Running: github_changelog_generator --user rails-on-services --project apartment --release-branch master --output CHANGELOG.md --token ***     
/usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator/generator/generator_fetcher.rb:88:in `block in associate_tagged_prs': No merge sha found for PR 97 via the GitHub API (StandardError)
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator/generator/generator_fetcher.rb:70:in `reject'
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator/generator/generator_fetcher.rb:70:in `associate_tagged_prs'
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator/generator/generator_fetcher.rb:50:in `add_first_occurring_tag_to_prs'
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator/generator/generator.rb:151:in `fetch_issues_and_pr'
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator/generator/generator.rb:47:in `compound_changelog'
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/lib/github_changelog_generator.rb:35:in `run'
	from /usr/local/bundle/gems/github_changelog_generator-1.15.0/bin/github_changelog_generator:5:in `<top (required)>'
	from /usr/local/bundle/bin/github_changelog_generator:23:in `load'
	from /usr/local/bundle/bin/github_changelog_generator:23:in `<main>'
```

**Comment by rpbaltazar** [comment]
I think it might be related with the fact that we're running the 1.15.0 instead of the latest 1.15.2

**Comment by charmixer** [comment]
Thanks for bringing this up.

I will try to upgrade to 1.15.2 and release version v1.1 of this action as soon as possible.

**Comment by charmixer** [comment]
I've updated to 1.15.2 and released v1.1 now: https://github.com/marketplace/actions/automatic-changelog-generator?version=v1.1

I have not tested your specific usecase, but if this doesn't fix your problem, then you probably need to open a issue with the developers of the changelog generator: https://github.com/github-changelog-generator/github-changelog-generator
